backend/src/services/bambora.service.ts

```diff
@@ -32,12 +32,15 @@ export class BamboraService {
 
   createBamboraPaymentRequest(paymentModel) {
     const amount = paymentModel.amount * 100;
-    const preTaxAmount = Math.round(
-      Number(
-        ((amount * Number(BAMBORA_TAX)) / (Number(BAMBORA_TAX) + 1)).toFixed(2),
-      ),
-    );
-    const taxAmount = BAMBORA_TAX;
+    const bamboraTax = Number(BAMBORA_TAX);
+    let preTaxAmount = amount;
+    if (bamboraTax) {
```

Comment:
try catch statements works better imo. You can also catch some other error in your code